domain/entity/Furnishing.ts

```diff
@@ -0,0 +1,15 @@
+import { Resources } from "./Resources";
+
+export class Furnishing {
```

Comment:
I don't remember the details, we want to separate Furnishing from just Tiles?

Reply:
Good question. Indeed I didn't checked the old plan. wdyt about having `Tile` as parent class of Furnishing?

Reply:
Hmm sure why not, we'll see